themes/ddbasic/scripts/ddbasic.common.js

```diff
@@ -172,8 +152,9 @@
       });
     });
 
+    // Hide empty element.
     $('.layout-wrapper').each(function() {
```

Comment:
Is this even needed? I removed "layout-wrapper" from our own layout too, because its not needed anymore: https://github.com/easySuite/easyddb_frontend_layout/commit/a45028628bc467b04d94f28c632743fb7f57508d.

However "layout-wrapper" is still used on pages with default layout like 404 page: https://www.faxebibliotek.dk/asdsadsa